src/player.cpp

```diff
@@ -2898,15 +2898,15 @@ void player::shout( std::string msg )
         base = 15;
         shout_multiplier = 3;
         if ( msg.empty() ) {
-            msg = _("yourself scream loudly!");
+            msg = is_player() ? _("yourself scream loudly!") : _("a loud scream!");
         }
     }
 
     if ( has_trait( trait_SHOUT3 ) ) {
         shout_multiplier = 4;
         base = 20;
         if ( msg.empty() ) {
-            msg = _("yourself let out a piercing howl!");
+            msg = is_player() ? _("yourself let out a piercing howl!") : _("a piercing howl!");
         }
     }
 
```

Comment:
One line down from here is the case for the shouter not having a mutation, missed that one.

Reply:
That's not critical 5hen, but might as well handle it.